products/extensions/subscriptions/concept.md

```diff
@@ -0,0 +1,29 @@
+# Concepts
+
+To use subscriptions, you will need to be familiar with two core concepts in subscription models.
+
+## Plans
+
+A plan is a set of rules that define the subscription. This includes the price, the billing interval, and the product that the customer will receive. Multiple intervals can be assigned to a single plan. Plans can be created and managed in the Shopware administration.
+
+## Intervals
+
+An interval is the time between each delivery cycle. For example, a delivery can be monthly, quarterly, or annually. Billing is triggered each time a delivery cycle repeats. The interval is defined in the plan and can be set to any time frame. This is also created and managed in the Shopware administration.
+
+Intervals can be of three different types:
+
+### Relative
+
+A relative interval is an interval that is determined by a previous interval. For example, if a customer subscribes to a monthly plan, the next interval will be one month after the first delivery. These intervals are determined by using PHP's `DateInterval` class.
+
+### Absolute
+
+An absolute interval is an interval that is determined by a fixed date. For example, if a customer subscribes to a monthly plan, the next interval will be on the same day of the month as the first delivery. These work with cron expressions.
```

Comment:
No, that's relative again, it's **_relative_** to the start date of the subscription. Absolut would be "the first of the month" or "Every Wednesday"